util/README.md

```diff
@@ -0,0 +1,22 @@
+# @polkassembly/util
+
+A set of utility functions used in [Polkassembly](https://polkassembly.io).
+
+## Usage
+
+Installation -
+
+```
+yarn add @polkssembly/util
+```
+
+Functions can be imported directly from the package, e.g.
+
+```js
+import { getPassingThreshold } from '@polkassembly/util';
```

Comment:
list all the functions in readme
{
  getPassingThreshold: [Function: getPassingThreshold],
  newtonRaphson: [Function: newtonRaphson],
  solveQuadraticEquation: [Function: solveQuadraticEquation]
}

Reply:
Hmm I don't get it, there is the list, L20 of this same README. Or did you mean something else?

Reply:
ah yes sorry